contracts/nexus-gateway/src/nexus.rs

```diff
@@ -1,8 +1,10 @@
 use axelar_wasm_std::nonempty;
-use connection_router::state::{Address, ChainName, Message as RouterMessage, ID_SEPARATOR};
+use connection_router::state::{
+    Address, ChainName, CrossChainId, Message as RouterMessage, ID_SEPARATOR,
```

Comment:
I don't like that we do this renaming: `Message as RouterMessage`. Just my take, but I feel like it makes the code confusing. Everywhere else it's referred to as `Message`, but in this crate we have a different type called `Message`, and I feel like that collision is a bit confusing when reading the code.

Reply:
What's your suggestion then?

Reply:
I would just rename `Message` in this file to `NexusMessage`

Reply:
I would re-export that `connection_router::state::Message` from `connection_router`.